instance/migrations/0048_appserver_refactor1.py

```diff
@@ -101,7 +100,6 @@ class Migration(migrations.Migration):
             options={
                 'verbose_name': 'Open edX Instance',
             },
-            bases=(instance.models.instance.Instance, models.Model),
```

Comment:
That's a gnarly one.. I had to do [something similar for a previous data migration](https://github.com/open-craft/opencraft/pull/175/commits/71e04d092512522279f7d838395ca4354cc4bdd5) in PR #175.